transform/mattermost-analytics/models/marts/data_eng/event_registry/fct_events_daily_snapshot.sql

```diff
@@ -0,0 +1,14 @@
+{{
+    config({
+        "tags":"hourly",
```

Comment:
why is it a `daily` snapshot (given that the tags value is hourly)?

Reply:
This tag is used to configure how often each model runs. Although it's a daily snapshot, it makes sense to update hourly so that we can see new events on the date they were added (with 1 hour delay top). 

If this is not needed, the tag can change to `nightly`.